x/examples/website-wrapper-app/wrapper_app_project/template/android/app/build.gradle.handlebars

```diff
@@ -1,10 +1,10 @@
 apply plugin: 'com.android.application'
 
 android {
-    namespace "org.getoutline.pwa"
+    namespace "{{appId}}"
     compileSdk rootProject.ext.compileSdkVersion
     defaultConfig {
-        applicationId "org.getoutline.pwa"
+        applicationId "{{appId}}.android"
```

Comment:
Why do you need .android? Is that best practice?
Having separate android, ios and macos ids have been a pain on outline